packages/slice-machine/lib/builders/SliceBuilder/Header/index.tsx

```diff
@@ -126,6 +148,13 @@ const Header: React.FC<{
             initialVariation={variation}
             variations={Model.variations}
           />
+          <RenameSliceModal
```

Comment:
I forgot if we had some thing to prevent more and one modal being open at a time,.... like the review modal

Reply:
<img width="755" alt="Screenshot 2022-06-27 at 12 21 21" src="https://user-images.githubusercontent.com/69536010/175930201-4923e236-8214-41ca-81c7-2d57abc09b95.png">
Are you referring to this?

Reply:
I'll need to dig into it, it's been awhile since I've worked with the modal mechanism :)